### PR TITLE
EM-75

### PR DESCRIPTION
**Pull request by yurige-matrix**
Finished

### CODE REVIEW

File: src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ const DiagnosisAndTreatment = () => {
             handlePopUpProps(
               findingsDetails,
               'findings_details',
-              'diagnosis_and_recommendations',
+              'diagnosi s_and_recommendations',
```
Comment (matrix-amiel) [inline src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js:79]: fix it

File: src/Components/Forms/MedicalAdmission/MedicalAdmission.js [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+  FormContext,
+  useForm,
+} from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import {
+  StyledForm,
+} from './Style';
+
+import RadioGroupChoice from 'Assets/Elements/RadioGroupChoice';
+import PopUpFormTemplates from '../../Generic/PopupComponents/PopUpFormTemplates';
```
Comment (matrix-amiel) [inline src/Components/Forms/MedicalAdmission/MedicalAdmission.js:15]: change to an absolute path

File: src/Components/Forms/MedicalAdmission/MedicalAdmission.js [hunk 3]
```diff
@@ -1,7 +1,22 @@
 //MedicalAdmission
-
 import { connect } from 'react-redux';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import VisitDetails from '../../Generic/PatientAdmission/PatientDetailsBlock/VisitDetails';
```
Comment (matrix-amiel) [inline src/Components/Forms/MedicalAdmission/MedicalAdmission.js:4]: change to an absolute path

File: src/Components/Forms/MedicalAdmission/MedicalAdmission.js (hunk 2 above)
Comment (yurige-matrix) [reply src/Components/Forms/MedicalAdmission/MedicalAdmission.js:15]: Fixed

File: src/Components/Forms/MedicalAdmission/MedicalAdmission.js (hunk 3 above)
Comment (yurige-matrix) [reply src/Components/Forms/MedicalAdmission/MedicalAdmission.js:4]: Fixed

File: src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js (hunk 1 above)
Comment (yurige-matrix) [reply src/Components/Forms/DiagnosisAndRecommendations/DiagnosisAndTreatment/index.js:79]: Fixed, thanks

**Pull request merged**